apps/openmw/mwclass/container.cpp

```diff
@@ -145,7 +145,7 @@ namespace MWClass
         // make key id lowercase
         std::string keyId = ptr.getCellRef().getKey();
         Misc::StringUtils::lowerCaseInPlace(keyId);
-        for (MWWorld::ContainerStoreIterator it = invStore.begin(); it != invStore.end(); ++it)
```

Comment:
Right now I could create the actual Iterator classes inheriting from Base, and then duplicate the constructors on them, because we can't inherit constructors without C++11. Or to not duplicate all contructors I could change some code in ContainerStore and create fewer constructors on the Iterator classes that takes the Base as argument, But I'm afraid that would ending with more calls to copy and slowing things down.

If you want, you could review as it is now and merge, and then I update my fork and solve this issue later with a better solution (maybe theres a template trickery I'm missing now) and I could also port all other parts of code which should be using the const iterator.